HW06.Rmd

```diff
@@ -19,10 +19,42 @@ Steps:
 
 From the RNA-Seq files, you only need the "Name" from one file and the "TPM" column from all the files. TPM stands for "transcripts per million" and is a common unit for normalized expression data. 
 ```{r load in data}
-#hint, using apply (specifically sapply) you can read in the data into a list and then bind the columns together. Or you can use a for loop too. 
-
-#you only need the 
-
+# load tidyverse 
+library(tidyverse)
+# list all the files
+h3k27m_vec <- list.files("RNA_Seq_processed/H3K27M")
+
+for (x in h3k27m_vec) {
+  if (x == h3k27m_vec[1]) {
+# read in the first file and select name and TPM
+    h3k27m <- read_tsv(paste("RNA_Seq_processed/H3K27M/", h3k27m_vec[1], sep = "")) %>% 
+            select("Name", "TPM")
+  } else {
+# read in and row bind each file to the former one and skip the first file 
+  h3k27m <- h3k27m %>% 
+    cbind(read_tsv(paste("RNA_Seq_processed/H3K27M/", x, sep = "")) %>% 
+# select TPM
+            select("TPM"))
+  }
+}
+# rename
+names(h3k27m) <- c("gene_id", paste("TPM.H3K27_Mutated.", 1:23, sep = ""))

```

Comment:
It's a great idea to name the TPM column after the H3K27M background and thereby classifying the type of cell line early on. This is especially useful later on when trying to compare between wt and H3K27M.